### ISSUE TITLE
[BUG] - Overseer bugged (Clean Install)

### ISSUE THREAD

**Issue by SmoofBrane** [Morph21/MercuryTrade-Community-Fork#31]
**Describe the bug**
The Overseer functionality does not work at all when using Vulkan or DX11.
Clicking "New > Icon" for example, does nothing, and the "Showcase" displays nothing.

**To Reproduce**
1. Download MercuryTrade .exe or .jar
2. Delete any pre-existing 'appdata/local' folders
3. Start MT
4. Go to 'Overseer settings'
5a. Switch to "Showcase" profile
5b. Click "New > Icon"

**Expected behavior**
a) See the preloaded showcase layout
b) Create a new "Icon" object in the default profile

**Screenshots**
No screenshots, but upon clicking anything in the "Overseer settings" menu gives a:
`ERROR ErrorHandler - java.lang.NoClassDefFoundError: com/sun/awt/AWTUtilities`
(My whole .log is abundantly full of them)

**Desktop (please complete the following information):**
 - Version 1.1.2

**Additional context**
I tried updating, uninstalling and reinstalling Java (8u271), removing the appdata/local folder,
exiting MT, exiting PoE, switching PoE renderers, restarting my pc.
Nothing has worked yet.

**.log File**
[mercury-app.log](https://github.com/Morph21/MercuryTrade-Community-Fork/files/5423537/mercury-app.log)

**Comment by SmoofBrane** [comment]
Small added bonus: I just switched back from DX11 to Vulkan, and now I'm stuck in a infinite loading loop.
At the very bottom of the log there are now four new entries.
One `ErrorHandler - null`, and three `ErrorHandler - java.lang.NoClassDefFoundError: com/sun/awt/AWTUtilities`.

New log: [mercury-app.log](https://github.com/Morph21/MercuryTrade-Community-Fork/files/5423621/mercury-app.log)

**Comment by Morph21** [comment]
It's probably caused by installing only java jfk not here. Try to install java from here https://java.com/download

**Comment by SmoofBrane** [comment]
> It's probably caused by installing only java jdk not jre. Try to install java from here https://java.com/download

Just uninstalled JDK and JRE, double checked with the Java Uninstall tool, reinstalled Java from the link (same exact file as I used before), and now it's working. Maybe there were some residual files or conflicting files between the old JDK and the new JRE, but removing JDk seemed to have done the trick.
Also, I did have JRE installed, the exact same file as you linked, and which was linked in the readme on this fork.
Weird, but it works now so I'm good. Thanks for the support and the dedication to this project.
